### ISSUE TITLE
Functions Request.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [fracte/keepnote#218]
```
Dear Matt Rasmussen,

Thank you very much for the release of version 0.6.2 with many bugs fixed.
Here I would like to request some new functions and I hope these can be
added to the later version of Keepnote.

1. Drug to import folders.
As I decide to use Keepnote for daily files arrangement, I found it was not
so easy to attach a lot of files. So it should be very ease if Keepnote
might import a folder/folders already exist.

2. Open the folder where the page located
Often, I should open the physical folder where a page located. Yet I can
not find such a function in Keepnote and I must open it use a file manager
such Nautilus and search the target file one by one manually. If the page
is featured with a right-click function to open the containing folder, it
will be very useful.

I know there were many people who requested different kinds of functions
and it is really difficult to meet the interest of everyone. But I still
hope my request be realized later at your convenience.

leon.yuxia


```

Original issue reported on code.google.com by `leon.yu...@gmail.com` on 22 Feb 2010 at 3:55

**Comment by GoogleCodeExporter** [comment]
```
Sorry, the type of this issue should be "Enhancement"
```

Original comment by `leon.yu...@gmail.com` on 22 Feb 2010 at 3:57

**Comment by GoogleCodeExporter** [comment]
```
Hi leon.yuxia,

isn't the 2. request "open the folder where the page is located" already 
implemented?

try menu: "view" -> "view note in file explorer"  or the same from the context 
menu
of the tree-view...

but mybe this function is only present in the windows version?

greetings...
```

Original comment by `nudels0...@googlemail.com` on 22 Feb 2010 at 8:13

**Comment by GoogleCodeExporter** [comment]
```
Hi  nudels0815,

The nautilus is used as the helper application of File Explorer, but I could not
"view note in file explorer" in my Ubuntu 9.10 Gnome 2.28.1 environment. 
Dolphin can
run but I usually use nautilus.

Thank you.
```

Original comment by `leon.yu...@gmail.com` on 23 Feb 2010 at 4:58

**Comment by GoogleCodeExporter** [comment]
```
sorry - I have only little experience with Linux so I can not help you with 
this :(
```

Original comment by `nudels0...@googlemail.com` on 27 Feb 2010 at 2:48

**Comment by GoogleCodeExporter** [comment]
```
Thank you anyhow! I installed Thunar, a file manager works in Xfce and Gnome 
similar
to Nautilus, and it can load the folder in Ubuntu 9.10. 
```

Original comment by `leon.yu...@gmail.com` on 27 Feb 2010 at 2:55

**Comment by GoogleCodeExporter** [comment]
```
Hi,

I believe feature 2 is supported with "View Note in File Explorer".  The 
application
you choose for File Explorer (see Edit > Preferences > Helper Applications) 
must be
able to take the full path of a folder as a single argument.  Therefore, if 
your file
manager of choice is not working (expects a different command line syntax), you 
could
write a shell script to wrap the file manager.

As for feature number 1, I think it is very interesting.  I myself would like 
that
feature, and may add it some time in the future.
```

Original comment by `matt.rasmus@gmail.com` on 1 Mar 2010 at 4:20
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Hi Matt,

Thank you very much for answering all my questions in detail. I have sloved 
these
according to your instructions. The other day I recommended Keepnote as a 
primary
document manager on http://forum.ubuntu.org.cn/ because Keepnote looks concise 
but
very easy to manage than other similar softwares. 

By the way, I read your CV and found you engaged in bioinformatic researchs and
published good quality articles. I am a molecular microbial researcher and I 
planned
to learn bioinformatics by myself from scratch. Hope you might give some help 
too
when I encounter problems :-). 

good day.

Leon
```

Original comment by `leon.yu...@gmail.com` on 1 Mar 2010 at 4:52

**Comment by GoogleCodeExporter** [comment]
```
Hi everyone,

The View Note in File Explorer was implemented very well with Nautilus(2.30.1) 
in the new testing version of Keepnote Linux edition (0.6.3; deb). The Chinese 
language has been embeded in this version. Yet there seems some uncompleted 
language translations existed.
```

Original comment by `leon.yu...@gmail.com` on 2 Jul 2010 at 7:37

Attachments:
- [Keepnote_0.6.3.png](https://storage.googleapis.com/google-code-attachments/keepnote/issue-218/comment-8/Keepnote_0.6.3.png)

**Comment by GoogleCodeExporter** [comment]
Original comment by `matt.rasmus@gmail.com` on 1 Nov 2011 at 2:43
- Changed state: **Done**
